monai/apps/detection/networks/retinanet_predictor.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, List, Optional
+
+import torch
+from torch import Tensor, nn
+
+from monai.inferers import SlidingWindowInferer
+
+
+def convert_dict_value_to_list(head_outputs: Dict[str, List[Tensor]]) -> Dict[str, List[Tensor]]:
```

Comment:
Should we make all these module functions in this function to be private with prefix "_"? I think they are only used by this `DictPredictor`?

Thanks.